### DISCUSSION TITLE
The CodePath property is missing

### DISCUSSION THREAD

**Discussion by yoann0723** [elgatosf/elgato#19] [Help] [answered]
I got this error message when execute DistributionTool if I set both codepathWin and CodePathMac property in the manifest.json
manifest.json:
![image](https://github.com/elgatosf/elgato/assets/13672219/26772683-67c7-40d9-b303-341b454f2570)

error message:
 "Error: The CodePath property is missing in the manifest.json file. Please add a valid "CodePath" property to the manifest.json."

Just setting "CodePath": "xxx.exe" works fine, but my plugin needs crossplatform.

**Comment by yoann0723** [comment, marked as answer]
I guess that I found the resason why that happend. that is both xxxplugin.exe and xxxplugin must exist at same time in the directory.

**Comment by ChekTek** [comment]
I believe this is a bug with the distribution tool, where it requires a "CodePath" even though you have provided both a "CodePathWin" and "CodePathMac". The distribution tool is soon to be end of life, in favor of better tooling! Apologies for the invonvenience.
